tests/pre-merge/unit/demos/test_predict_video.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import os
+import shutil
+import sys
+
+import cv2
+import ffmpeg
+import pytest
+
+from geti_sdk.demos import predict_video_on_local
+
+
+class TestPredictVideo:
+    @pytest.mark.parametrize("preserve_audio", [True, False])
+    def test_predict_video_on_local(
```

Comment:
Could you give an indication of the execution time for this test?

Reply:
It depends on the system. It takes about 10 sec on my machine(i9-11900). Do you think I need to cut the test video?

Reply:
No, that's fine. The CI tests run on an i9 machine as well so it should be no problem